src/sentry/api/endpoints/project_ownership.py

```diff
@@ -79,6 +82,21 @@ def save(self):
 
         return ownership
 
+    def __modify_auto_assignment(self, ownership):
+        import logging
+
+        logger = logging.getLogger(__name__)
+        changed = False
+        logger.info(self.object)
```

Comment:
Why are you logging the entire object?

Reply:
This is temporary code, used for debugging. I intended to remove it for the final review.